sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -54,30 +54,36 @@ import {
   setStrEncodingParam,
   StringIndexType
 } from "./util";
-import { BeginAnalyzeHealthcarePoller, HealthcarePollerLike } from "./lro/health/poller";
+import {
+  BeginAnalyzeHealthcarePoller,
+  AnalyzeHealthcareEntitiesPollerLike
+} from "./lro/health/poller";
```

Comment:
just curious about if the namespace "./lro/health/poller" matters in the JS/TS.  What want to learn what it is

Reply:
this is not a name space, it is just a module.